dask/dataframe/io/io.py

```diff
@@ -232,6 +232,7 @@ def from_pandas(data, npartitions=None, chunksize=None, sort=True, name=None):
     return new_dd_object(dsk, name, data, divisions)
 
 
+@_deprecated()
```

Comment:
Small suggestion to add the version of `dask` after which `from_bcolz` was deprecated. See this comment https://github.com/dask/dask/pull/8752#discussion_r813389574 for more details. 

```suggestion
@_deprecated(after_version="2022.02.0")
```

Reply:
Thank you!